doc/nvim-tree-lua.txt

```diff
@@ -291,6 +291,7 @@ Subsequent calls to setup will replace the previous configuration.
         enable = false,
         show_on_dirs = false,
         debounce_delay = 50,
+        severity = null,
```

Comment:
Please run update-help.sh as per https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md

This will set the default value.

Reply:
Done